Rakefile

```diff
@@ -21,10 +21,14 @@ rescue LoadError
   # no rspec available
 end
 
-def build(os, org="intelsdi")
+def build(os, org="intelsdi", version=nil)
   raise "Error: missing operating system" unless os
   raise "Error: invalid operating system #{os}" unless File.exists? "#{os}/Dockerfile"
-  sh "docker build -t #{org}/snap:#{os} --build-arg BUILD_DATE=$(date +%Y-%m-%d) -f #{os}/Dockerfile ."
+  if version
+    sh "docker build -t #{org}/snap:#{version}_#{os} --build-arg BUILD_DATE=$(date +%Y-%m-%d) --build-arg SNAP_VERSION=#{version} -f #{os}/Dockerfile ."
+  else
+    sh "docker build -t #{org}/snap:#{os} --build-arg BUILD_DATE=$(date +%Y-%m-%d) -f #{os}/Dockerfile ."
```

Comment:
It depends on the container Dockerfile, but yes in general latest:
https://github.com/intelsdi-x/snap-docker/blob/master/alpine/Dockerfile#L5